### PR TITLE
Crm updates

### PR DESCRIPTION
**Pull request by kevinhowbrook**
- [x] Build out the constituent profile more (phone, address etc)

Note: this is mainly a POC so I'm not paying too much attention to validation but if there is time I will.

### CODE REVIEW

File: npr_poc/forms/models.py [hunk 1]
```diff
@@ -57,9 +57,29 @@ class FormPage(WagtailCaptchaEmailForm, BasePage):
...
+    phone_number_type = forms.ChoiceField(choices=[
+        ("Mobile","Mobile"),
+        ("Home","Home"),
+        ("Business","Business"),
+    ])
+    phone_consent = forms.BooleanField(label="May we contact you by phone?", required=False)
+    address_line_one = forms.CharField(label="Address line one, E.G 123 Main street")
+    city = forms.CharField(label="City")
+    country = forms.CharField(label="County")
+    county = forms.CharField(label="county")
```
Comment (tomdyson) [inline npr_poc/forms/models.py]: county before country?

Comment (kevinhowbrook) [reply npr_poc/forms/models.py]: fixed in https://github.com/torchbox/npr-poc/pull/36/commits/d14d8581f82f297d270e91290fb49c0e59644884

File: npr_poc/forms/models.py [hunk 2]
```diff
@@ -57,9 +57,29 @@ class FormPage(WagtailCaptchaEmailForm, BasePage):
...
+    phone_number = forms.CharField(label="Phone number")
+    phone_number_type = forms.ChoiceField(choices=[
+        ("Mobile","Mobile"),
+        ("Home","Home"),
+        ("Business","Business"),
+    ])
+    phone_consent = forms.BooleanField(label="May we contact you by phone?", required=False)
+    address_line_one = forms.CharField(label="Address line one, E.G 123 Main street")
+    city = forms.CharField(label="City")
+    country = forms.CharField(label="County")
```
Comment (tomdyson) [inline npr_poc/forms/models.py]: label -> Country

File: npr_poc/forms/models.py (hunk 1 above)
Comment (tomdyson) [inline npr_poc/forms/models.py]: county->County

**Pull request merged**